src/scarlett_mixer.c

```diff
@@ -1315,8 +1316,8 @@ static RobWidget* toplevel (RobTkApp* ui, void* const top) {
 
 	/* output level + labels */
 	for (unsigned int o = 0; o < ui->device->smst; ++o) {
-		int row = 4 * floor (o / 5); // beware of bleed into Hi-Z, Pads
-		int oc = o % 5;
+		int row = 0; // beware of bleed into Hi-Z, Pads
+		int oc = o;
```

Comment:
This breaks layout for the 18i20 -- rows have to be split after 5 stereo pairs,  see  27df030d17c
https://github.com/x42/scarlett-mixer/issues/6#issuecomment-362452466
https://github.com/x42/scarlett-mixer/issues/6#issuecomment-362715456

Reply:
this one is also interesting -- can we somehow keep both layouts?
It splits Headphone 2R, S/PDIF L,R to row 4 and overlaps with the pads.